### ISSUE TITLE
Deprecated nerdfont icons

### ISSUE THREAD

**Issue by ImPrankster** [CosmicNvim/CosmicNvim#103]
With a newer version of nerd font, some of the icons will not render correctly. e.g.
<img width="205" alt="Screenshot 2023-07-12 at 13 52 02" src="https://github.com/CosmicNvim/CosmicNvim/assets/41331355/e6cf0a0b-74a7-45a2-a531-fa48962a2019">
using `CaskaydiaCove Nerd Font`

**Comment by mattleong** [comment]
I don't believe this would be a CosmicNvim issue 🤔 Maybe an issue with `nvim-tree/nvim-web-devicons`?

**Comment by ImPrankster** [comment]
I think this has something to do with icons defined in https://github.com/CosmicNvim/CosmicNvim/blob/main/lua/cosmic/utils/icons.lua
are deprecated in newer nerd font versions.
